osm2gtfs.py

```diff
@@ -82,28 +92,28 @@ def main():
 
     schedule = transitfeed.Schedule()
     agency = schedule.AddAgency(
```

Comment:
Another approach is in my fork, I which I add a method that returns a *transitfeed.Agency* object created using info from a json file (input_incofer.json), in the json file the valid keys are any of the *transitfeed.Agency._FIELD_NAMES* array values.